gcmr/assembly/steady_state/Griffin_steady_state.i

```diff
@@ -0,0 +1,214 @@
+                         803 802 810 806 807 807 804 801 801 808 809 805 805'
+  []
+  [coarse_mesh]
+    type = GeneratedMeshGenerator
+    dim= 3
+    nx = 10
+    ny = 10
+    nz = 20
+    xmin = -0.1
+    xmax =  0.1
+    ymin = -0.1
+    ymax =  0.1
+    zmin =  0.
+    zmax =  2.
+  []
+  [assign_coarse_id]
+    type = CoarseMeshExtraElementIDGenerator
+    input = fmg_id
+    coarse_mesh = coarse_mesh
+    extra_element_id_name = coarse_element_id
+  []
+[]
+
+[Executioner]
+  type = SweepUpdate
+
+  richardson_abs_tol = 1e-8
+  richardson_rel_tol = 1e-8
+  richardson_value = eigenvalue
+
+  richardson_max_its = 1000
+  inner_solve_type = GMRes
+  max_inner_its = 20
+
+  fixed_point_max_its = 1
+  custom_pp = fission_source_integral
+  custom_rel_tol = 1e-6
+  force_fixed_point_solve = true
+
+  cmfd_acceleration = true  #false
+  coarse_element_id = coarse_element_id
+  cmfd_eigen_solver_type = newton
+  prolongation_type = multiplicative
+  max_diffusion_coefficient = 1
+[]
+
+[Debug]
+  check_boundary_coverage = false
+  print_block_volume = false
+  show_actions = false
+[]
+
+[AuxVariables]
+  [Tf]
+    initial_condition = 873.15
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+
+[TransportSystems]
+  particle = neutron
+  equation_type = eigenvalue
+
+  G = 11
+  VacuumBoundary = '2000 2001'
+  ReflectingBoundary = '9000'
+
+  [SN]
+    scheme = DFEM-SN
+    family = MONOMIAL
+    order = FIRST
+
+    AQtype = Gauss-Chebyshev
+    NPolar = 2
+    NAzmthl = 3
+    NA = 2
+    n_delay_groups = 6
+
+     sweep_type = asynchronous_parallel_sweeper
+     using_array_variable = true
+     collapse_scattering  = true
+     hide_angular_flux = true
+  []
+[]
+
+[GlobalParams]
+  library_file = '../ISOXML/XS_Griffin.xml'
+  library_name = XS_Griffin
+  isotopes = 'pseudo'
+  densities = 1.0
+  is_meter = true
+# power normalization
+  plus = true
+  dbgmat = false
+  grid_names = 'Tmod'
+  grid_variables = 'Tf'
+[]
+
+[PowerDensity]
+  power = 225e3 # Assembly Power from NS
+  power_density_variable = power_density
+  integrated_power_postprocessor = integrated_power
+[]
+
+[Materials]
+  [mod]
+    type = CoupledFeedbackMatIDNeutronicsMaterial
+    block = '10 100 102 103 200 201 300 400 401 500 600 8000 8001'
+    # TODO: Add changes in densities
+  []
+[]
+
+[MultiApps]
+  [bison]
+    type = FullSolveMultiApp
+    positions = '0 0 0'
+    input_files  = BISON.i
+    execute_on = 'timestep_end'
+    # no_backup_and_restore = true # to restart from the latest solve of the multiapp (for pseudo-transient)
+    keep_solution_during_restore = true
+    #app_type = SamApp
+    #library_name = libsam-opt.la
+	#library_path = '/beegfs1/software/NEAMS_microreactor/projects_super_mar22/SAM/lib'
+  []
+[]
+
+[Transfers]
+  [to_sub_power_density]
+    type = MultiAppProjectionTransfer
+    direction = to_multiapp
+    multi_app = bison
+    variable = power_density
+    source_variable = power_density
+    execute_on = 'initial timestep_end'
```

Comment:
Why do we need to have the transfer on initial? If not, we can remove this parameter because the default value of a transfer is the same execute_on as the multiapp.

Reply:
I changed it to  execute_on = 'timestep_end'.